torch/csrc/jit/codegen/cuda/kernel_ir_builder.cpp

```diff
@@ -121,6 +121,20 @@ Int* IrBuilder::one() {
   return one_;
 }
 
+Bool* IrBuilder::falseVal() {
+  if (false_ == nullptr) {
+    false_ = create<kir::Bool>(false);
+  }
+  return false_;
+}
+
+Bool* IrBuilder::trueVal() {
+  if (true_ == nullptr) {
+    true_ = create<kir::Bool>(true);
+  }
+  return true_;
+}
```

Comment:
Can we use consistent naming? Either changing `one()` and `zero()` to `oneVal()` and `zeroVal()` or changing `falseVal()` and `trueVal()` to `false()` and `true()` would be more consistent. Either option seems fine to me.